manifoldco/ui
# convert manifold-data-resource-list to gql

## Reason for change



- convert `manifold-data-resource-list` to GraphQL

## Testing


- In Storybook, with a valid token set, go to "Resource List 🔒 [Data]"
- Check that the resources are being loaded.
- Check that the links are set properly and the event triggered has the correct data

## Checklist




[docs]: https://ui.sandbox.manifold.co
[platform-testing]: https://app.gitbook.com/@manifold/s/engineering/playbooks/testing-your-code-against-a-platform
